docs/simulation.md

```diff
@@ -191,7 +191,7 @@ for the binary expansion of the passed integer.
 
 The xmon simulator is designed to work with operations that are either a ``GateOperation`` applying an ``XmonGate``,
 a ``CompositeOperation`` that decomposes (recursively) to ``XmonGates``,
-or a 1-qubit or 2-qubit operation with a ``KnownMatrix``.
+or a 1-qubit or 2-qubit operation with a known unitary matrix (``has _unitary_``).
```

Comment:
Maybe: "a 1-qubit or 2-qubit that implements the `_unitary_` method to return the known unitary matrix for the operation."

Reply:
Done.